server/src/listeners/views/modalView.ts

```diff
@@ -0,0 +1,136 @@
+type StandupModalArguments = {
+  client: {
+    chat: { postMessage: Function };
+    users: { profile: { get: Function } };
+    token: string;
+  };
+  context: { botToken: string };
+  ack: Function;
+  payload: { channel_id: string; state: { values: any | object } };
+  body: { user: { id: string } };
+};
+
+const standupModal = async ({
+  ack,
+  client,
+  payload,
+  context,
+  body,
+}: StandupModalArguments) => {
```

Comment:
Optional: As in frontend we usually name it `ComponentProps`
```suggestion
}: StandupModalArgs) => {
```